portfolio/src/main/webapp/style.css

```diff
@@ -1,9 +1,56 @@
+html{
+    background-color: rgb(73, 67, 67);
+    color: #f2f2f2;
+}
+
+body{
+    max-width: 800px;
+    margin-left: auto;
+    margin-right: auto;
+}
+
+#navbar .navbar-item{
+    padding:8px 16px;
+    float:left;
+    width:auto;
+    border:none;
+    display:block;
+    outline:0s;
+}
```

Comment:
nit: alphabetize the fields 
e.g.:
border
display
float
....

Reply:
> nit: alphabetize the fields
> e.g.:
> border
> display
> float
> ....

I made these changes in PR #6